README.md

```diff
@@ -13,3 +13,25 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+
+Part 1:
+1. Depending on the memory the machine has, or the program acquires, this can slow runtime. For example, if your program uses up a lot of memory, this can slow things down. Like if you had a bunch of arrays, strings, etc your program was looking through and there was not a way to get rid of memory you don't need, this can cause issues down the line. While this may not seem like a cause, while asymptotic complexity can tell you how much space your program will take, it cannot see the other programs running in the background. If RAM does not have enough storage to run this program, it may lag and have a longer runtime becuase it is waiting for memory to be freed up. 
```

Comment:
I still don't see how this means that asymptotic analysis can be misleading, and you're not giving any specific examples, only lots of things that may happen in some way.